systemtest/src/main/java/io/strimzi/systemtest/utils/TestKafkaVersion.java

```diff
@@ -204,4 +204,8 @@ public static TestKafkaVersion getSpecificVersion(String kafkaVersion) {
     public static TestKafkaVersion getSpecificVersionFromList(List<TestKafkaVersion> versions, String kafkaVersion) {
         return versions.stream().filter(it -> it.version.equals(kafkaVersion)).collect(Collectors.toList()).get(0);
     }
+
+    public static TestKafkaVersion getLatestSupportedKafkaVersion() {
+        return supportedKafkaVersions.get(supportedKafkaVersions.size() - 1);
+    }
```

Comment:
I guess that `getDefaultSupportedKafkaVersion()` does the same thing. "Latest" Kafka version in `kafka-versions.yaml` is labeled with `default: true` IIRC

Reply:
there is no use for getting latest since i use ``` this.olmUpgradeData.setProcedures(new UpgradeKafkaVersion(TestKafkaVersion.getDefaultSupportedKafkaVersion())); ``` in the code. so this is a mistake. thanks for noticing